.env_app-example

```diff
@@ -15,3 +15,5 @@ export BASE_URL="http://localhost:8000/"
 # Make sure APP_ROOT variable ends with a /
 # this is the prefix that vogon will be deployed under (e.g. "vogon-2" for deployment at my-server.org/vogon-2)
 export APP_ROOT=
+
+export QUADRIGA_ENDPOINT=
```

Comment:
this needs to have an example (as a comment), e.g http://quadriga.instance/quadriga/...whatever else needs to be here/